Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/CancelDeletionProcessDuringGracePeriod/HandlerTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
+using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletionProcessDuringGracePeriod;
+using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using FakeItEasy;
+using FluentAssertions;
+using Xunit;
+using static Backbone.UnitTestTools.Data.TestDataGenerator;
+using Handler = Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletionProcessDuringGracePeriod.Handler;
+
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.Identities.Commands.CancelDeletionProcessDuringGracePeriod;
+public class HandlerTests
+{
+    [Fact]
+    public async Task Happy_path()
+    {
+        // Arrange
+        var activeIdentity = TestDataGenerator.CreateIdentityWithApprovedDeletionProcess(SystemTime.UtcNow.Date);
+        var activeDevice = activeIdentity.Devices[0];
+
+        var mockIdentitiesRepository = A.Fake<IIdentitiesRepository>();
+        var fakeUserContext = A.Fake<IUserContext>();
+
+        A.CallTo(() => mockIdentitiesRepository.FindByAddress(A<IdentityAddress>._, A<CancellationToken>._, A<bool>._))
```

Comment:
I would be more specific here, i.e. instead of `A<IdentityAddress>._` I would use `activeIdentity.Address`. Also, I think we agreed earlier that `CancellationToken.None` should be used instead of `A.CancellationToken>._`.